src/patternfly/components/Hint/hint.scss

```diff
@@ -0,0 +1,84 @@
+.pf-c-hint {
+  --pf-c-hint--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-hint--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-hint--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-hint--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-hint--BackgroundColor: var(--pf-global--palette--blue-50);
+  --pf-c-hint--BorderColor: var(--pf-global--palette--blue-100);
+  --pf-c-hint--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-hint--BoxShadow: var(--pf-global--BoxShadow--sm);
+  --pf-c-hint--Color: var(--pf-global--Color--100);
+
+  // Hint Title
+  --pf-c-hint__title--FontSize: var(--pf-global--FontSize--lg);
+
+  // Hint Body
+  --pf-c-hint__body--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-hint__title--body--MarginTop: var(--pf-global--spacer--md);
+
+  // Hint Footer
+  --pf-c-hint__footer--child--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-hint__title--footer--MarginTop: var(--pf-global--spacer--md);
+  --pf-c-hint__body--footer--MarginTop: var(--pf-global--spacer--md);
+
+  // Hint Actions
+  --pf-c-hint__actions--MarginLeft: var(--pf-global--spacer--2xl);
+  --pf-c-hint__actions--c-dropdown--MarginTop: calc(var(--pf-global--spacer--form-element) * -1);
+
+  display: grid;
+  grid-template-columns: 1fr auto;
+  padding: var(--pf-c-hint--PaddingTop) var(--pf-c-hint--PaddingRight) var(--pf-c-hint--PaddingBottom) var(--pf-c-hint--PaddingLeft);
+  color: var(--pf-c-hint--Color);
+  background-color: var(--pf-c-hint--BackgroundColor);
+  border: var(--pf-c-hint--BorderWidth) solid var(--pf-c-hint--BorderColor);
+  box-shadow: var(--pf-c-hint--BoxShadow);
+
+  .pf-c-button.pf-m-link.pf-m-inline {
+    text-align: left;
+    white-space: pre-wrap;
+  }
+}
+
+.pf-c-hint__actions {
+  margin-left: var(--pf-c-hint__actions--MarginLeft);
+  text-align: right;
+  grid-column: 2;
+  grid-row: 1 / span 3;
+
+  .pf-c-dropdown {
```

Comment:
This should only apply to plain dropdowns, in case someone puts a regular dropdown there.